inc/streams/LevelDetectorSPL.h

```diff
@@ -0,0 +1,143 @@
+/*
+The MIT License (MIT)
+
+Copyright (c) 2017 Lancaster University.
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+DEALINGS IN THE SOFTWARE.
+*/
+
+#include "CodalConfig.h"
+#include "DataStream.h"
+
+#ifndef LEVEL_DETECTOR_SPL_H
+#define LEVEL_DETECTOR_SPL_H
+
+
+/**
+  * Sensor events
+  */
+#define SPL_LEVEL_THRESHOLD_LOW                           1
+#define SPL_LEVEL_THRESHOLD_HIGH                          2
```

Comment:
I wonder about using the same defines as the normal level detector, i.e. remove these and use `LEVEL_THRESHOLD_LOW` and `LEVEL_THRESHOLD_HIGH` in your code. Makes the end user code more consistent.